widget/src/providers/ConfigProviders.tsx

```diff
@@ -0,0 +1,31 @@
+import { Chain, Token } from '@velocitylabs-org/turtle-registry'
+import { createContext } from 'react'
+
+export interface ConfigRegistryType {
+  chains: Chain['uid'][]
+  tokens: Token['id'][]
+}
+
+export const ConfigContext = createContext<{
+  allowedChains: Chain['uid'][]
+  allowedTokens: Token['id'][]
+}>({
+  allowedChains: [],
+  allowedTokens: [],
+})
```

Comment:
```suggestion
export const ConfigContext = createContext<{
  chains: Chain['uid'][]
  tokens: Token['id'][]
}>({
  chains: [],
  tokens: [],
})
```

Is there no way to not have to repeat the fields?

Reply:
what you mean? 🤔